scripts/contentScript.js

```diff
@@ -380,6 +380,99 @@ async function FilesInPrBitbucket(response) {
 	});
 }
 
+const githubHunkHighlight = async (apiResponses) => {
+
+	// Todo : optomization needed to not highlight next deleted line space if not present in response.
+	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
+	getFileName.forEach(async (item) => {
+		let FileContent = item.getAttribute('data-tagsearch-path');
+		if (FileContent) {
+
+			let matchEncrypted = await sha256(FileContent);
+			let foundFile = null;
+			for (k in apiResponses["hunkinfo"]) {
+				let item = apiResponses["hunkinfo"][k];
+				if (item.file === matchEncrypted) {
+					foundFile = item;
```

Comment:
Shouldn't we `break` after finding the file? If not, is it possible to find another item that matches this? If yes, then what happens to the previously found item?